src/ovs/aca_ovs_l2_programmer.cpp

```diff
@@ -134,9 +134,6 @@ int ACA_OVS_L2_Programmer::setup_ovs_bridges_if_need()
     execute_openflow_command("add-flow br-tun \"table=20,priority=1 actions=resubmit(,22)\"",
                              not_care_culminative_time, overall_rc);
 
-    execute_openflow_command("add-flow br-tun \"table=2,priority=25,arp,arp_op=1,in_port=\"patch-int\" actions=resubmit(,51)\"",
-                             not_care_culminative_time, overall_rc);
-
```

Comment:
please go ahead and remove the rule below also, since table=51 is for static ARP responder only and it is no longer needed.
execute_openflow_command("add-flow br-tun \"table=51,priority=1 actions=resubmit(,22)\"",
                         not_care_culminative_time, overall_rc);